### PR TITLE
Mobile version skeleton - First pull request

### PR DESCRIPTION
**Pull request by ForHemer**
The goal of this milestone is to create the **skeleton of my portfolio website.** This feature has a homepage with:
-  A logo
-  A menu
- The content
- And some social media to stay in touch

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                I’m a software developer! I can help you build a product, feature or website Look through some of my work
+                and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+            <a class="connect">LET’S CONNECT</a>
+            <ul class="social_media">
+                <li><a href="http://twitter.com/HemersonForeste"><img src="images/bird.svg" alt="twitter"></a></li>
+                <li><a href="https://www.linkedin.com/in/hemerson-foreste-890685197/"><img src="images/Linkedin.png" alt="Linkedin"></a></li>
+                <li><img src="images/m.svg" alt=""></li>
+                <li><img src="images/cat.svg" alt=""></li> 
+                <li><img src="images/hand.svg" alt=""></li>
```
Comment (Rustamxon7) [inline index.html:28]: - [ ] You used `<a>` links first 2 images, why dont use it also here. Please kindly fix this issue, you just have to put theme `` tags inside `<a>` tag 😃👌

File: style.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
+/* Import Poppins font in the project */
+@font-face {
+  font-family: 'Poppins';
+  src: url(Poppins/Poppins-Bold.ttf);
+}
+
+/* Background image on the body element */
+body {
+  background-image: url(images/Background.svg);
+  background-repeat: no-repeat;
```
Comment (Rustamxon7) [inline style.css:10]: - [ ] I would recommend removing this background from body and adding to `section` element. Please cut this 2 lines and in css add them into `section` element like this 😊🙌: 
```css
section {
   background-image: url(images/Background.svg);
   background-repeat: no-repeat;
}
```

**Review by Rustamxon7 — CHANGES_REQUESTED** [review]
> ## STATUS: CHANGES REQUIRED 🟡
> Hi @ForHemer ,
> 
> Great work on making the changes requested by the previous reviewer, but there are still some which aren't addressed yet.
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> really close
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ForHemer) [reply index.html:28]: Because the previous reviewer told me to change them
Screenshot from 2021-11-30 13-20-44
.

File: style.css (hunk 2 above)
Comment (ForHemer) [reply style.css:10]: If I put the **"background-image"** into the section part, my work will be different from the template.

File: index.html (hunk 1 above)
Comment (ForHemer) [reply index.html:28]: Because I don't know what link to use for them.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (MrBrN197) [inline .gitignore:1]: - [ ] [OPTIONAL] You can put your node_modules/ folder into this .gitignore file.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        <link rel="stylesheet" type="text/css" href="style.css">
+        <title>Personal Portfolio</title>
+    </head>
+    <body>
+        <header>
+            <a href="index.html"><img class="logo" src="images/log.png" alt="logo" width="30"></a>
+            <nav><img class="menu" src="images/Union.png" alt="menu"></nav>
+        </header>
+        <section>
+            <h1>I’m Hemerson Glad to see you!</h1>
```
Comment (MrBrN197) [inline index.html:17]: - [ ] [OPTIONAL] Well done on your current design. Even though the previous reviewer never mentioned this you should always keep a close attention to the colors and fonts when you are trying to match a template design. notice how your font color is black but in the Figma template it should be the color value `#172B4D`.
## Your Design 
image

## Figma Design
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+            <p class="information">
+                I’m a software developer! I can help you build a product, feature or website Look through some of my work
+                and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+            <a class="connect">LET’S CONNECT</a>
```
Comment (MrBrN197) [inline index.html:22]: - [ ] [OPTIONA] Again your design is really great 👏. As I said earlier please try to always keep an eye out for small features like font-weights. here, your "Let's Connect" text is too bold compared to the Figma template please adjust this.
## Your Design
image
## Figma Design
image

**Review by MrBrN197 — APPROVED** [review]
## Hi, @ForHemer  👏

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**